lab-doug/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-doug",
+  "version": "1.0.0",
+  "description": "cowsay homework",
+  "main": "server.js",
+  "scripts": {
+    "test": "mocha",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gsmatth/lab-06-cowsay-http-server.git"
+  },
+  "author": "gsmatth@icloud.com",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/gsmatth/lab-06-cowsay-http-server/issues"
+  },
+  "homepage": "https://github.com/gsmatth/lab-06-cowsay-http-server#readme",
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "cowsay": "^1.1.6",
```

Comment:
Dev dependencies are those that are only needed for developers working on writing the code. Regular dependencies are ones that are required for the code to work for anyone. Cowsay is the main content generator, so it should be a normal dependency, not a dev dependency.